network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/cohort/NetpCohortPixelInterceptor.kt

```diff
@@ -82,6 +82,8 @@ class NetpCohortPixelInterceptor @Inject constructor(
             "m_netp_ev_waitlist_notification_launched",
             "m_netp_ev_waitlist_enabled",
             "m_netp_ev_terms_accepted",
+            "m_netp_imp_geoswitching",
```

Comment:
Adding here since you can interact with geoswitching without enabling netp (if we decide to remove autoenable)